### ISSUE TITLE
Datadog API Key exposed in plaintext 

### ISSUE THREAD

**Issue by shibayu36** [auth0/auth0-deploy-cli#601]
### Describe the problem you'd like to have solved
I found a secret is exported in log-streams setting.  For example, if we set up a log stream for Datadog,  then datadogApiKey is exported.

```json
{
  "name": "Datadog",
  "type": "datadog",
  "status": "active",
  "sink": {
    "datadogApiKey": "...",
    "datadogRegion": "us"
  }
}
```

I don't want to dump a secret into a file because it increases the risk of leakage.

### Describe the ideal solution
So far, a secret has not been dumped by auth0-deploy-cli, and we use the management console to set this. for example

* configuration for Rules
* secrets for Actions

I think log-stream setting should be the same as these.

## Alternatives and current work-arounds

> A clear and concise description of any alternatives you've considered or any work-arounds that are currently in place.

### Additional context

> Add any other context or screenshots about the feature request here.

**Comment by qbunt** [comment]
Just found this as well. This has been open since July of '22, is there a team working on this at all? For a security related issue, in a security adjacent company, this seems like misstep.

**Comment by willvedd** [comment]
**Update: ** Again, I admit that this is a real security issue and could cause trouble for unsuspecting users. I've escalated this to our security team, who are interested in getting this addressed. I've also recategorized this as a bug (despite this being an API limitation). Don't have any additional details though, but will update as I get them. Hoping to have this addressed soon!

**Comment by willvedd** [comment]
Fixed with [v7.17.5](https://github.com/auth0/auth0-deploy-cli/releases/tag/v7.17.5). Sensitive log streams keys will now be obfuscated similarly to other secrets. Thanks for everyone's patience here!
